pkg/step/feature/derivation.go

```diff
@@ -486,6 +487,23 @@ func setBackToIR(trainStmt *ir.TrainStmt, fcMap ColumnMap, columnTargets []strin
 	}
 }
 
+// setBackToLabel update label field meta
+func setBackToLabel(trainStmt *ir.TrainStmt, fmMap FieldDescMap) {
```

Comment:
I am not sure what does the phrase *set back* mean?

Reply:
Indeed "set back" is not easy to understand. I was thinking, this function is setting derived column information back to the original IR structure, so it's a "set back", maybe we need a better name. I am updating this to the comment though.

Reply:
Do you mean `updateLabel` or `deriveLabel` like `deriveColumn`?

Reply:
I prefer `deriveLabel` will update this in the next PR.